Mathlib/LinearAlgebra/Determinant.lean

```diff
@@ -181,6 +180,7 @@ protected irreducible_def det : (M →ₗ[A] M) →* A :=
   else 1
 #align linear_map.det LinearMap.det
 
+open Classical in
```

Comment:
These ones are unavoidable, but this makes sure that we're honest about the scope